### DISCUSSION TITLE
Option to Display Tasks in All Spaces

### DISCUSSION THREAD

**Discussion by JosTheDude** [lawand-dot-io/taskbar#112] [Feature requests]
I was wondering if it would be possible to make Taskbar show all of the tasks open, regardless of space. I migrated from Hyprland to MacOS and am very very used to multi-workspace setups so would be excellent to be able to see everything from the taskbar.

**Comment by lawand-dot-io** [comment]
Thanks for the suggestion :) This a duplicate of https://github.com/lawand-dot-io/taskbar/discussions/39 so make sure the upvote that one. Cheers
